### PR TITLE
Add a new evolution UI

### PR DESCRIPTION
**Pull request by psykzz**
## About The Pull Request
Adds a new UI for xenos when evolving

## Why It's Good For The Game
Better UI for xenos with more knowledge over what your doing when you evolve

## Changelog
:cl:
add: new evolve screen
/:cl:

image

### CODE REVIEW

**Comment by psykzz** [comment]
As in after that choice?
So runner would show 
----
Runner

Evolves into
 - Hunter
 - Ravager

**Comment by psykzz** [comment]
A note this had an issue where only the first xeno could evolve.

**Comment by spookydonut** [comment]
@psykzz like this 
image

**Comment by spookydonut** [comment]
@psykzz update screenshots?

**Comment by psykzz** [comment]
image

File: tgui/packages/tgui/interfaces/HiveEvolveScreen.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    .filter(ability => filteredAbilities.indexOf(ability.name) === -1);
+  const lastItem = Object.keys(props.abilites).slice(-1)[0];
+
+  return (
+    <Section height={"100%"} title={props.name}>
+      <strong>Abilities</strong>
+      <span style={{ color: 'grey' }}>{' (hover for details)'}</span>
+      <p>
+        {props.abilites ? (
+          <ul>
```
Comment (spookydonut) [inline tgui/packages/tgui/interfaces/HiveEvolveScreen.js:32]: use LabeledList

File: tgui/packages/tgui/interfaces/HiveEvolveScreen.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+const CasteView = props => {
+  const filteredAbilities = ["Rest", "Regurgitate"];
+  const abilites = Object
+    .values(props.abilites)
+    .filter(ability => filteredAbilities.indexOf(ability.name) === -1);
+  const lastItem = Object.keys(props.abilites).slice(-1)[0];
+
+  return (
+    <Section height={"100%"} title={props.name}>
+      <strong>Abilities</strong>
```
Comment (spookydonut) [inline tgui/packages/tgui/interfaces/HiveEvolveScreen.js:28]: `<Section level={2}>`

**Comment by psykzz** [comment]
image

**Pull request merged**